src/main/java/uk/gov/hmcts/reform/sscs/service/SscsPdfService.java

```diff
@@ -85,6 +86,22 @@ public SscsCaseData generatePdf(SscsCaseData sscsCaseData, Long caseDetailsId, S
                 placeholders.put("welsh_exclude_dates",welshExcludesDates);
             }
             placeholders.put("welshCurrentDate",LocalDateToWelshStringConverter.convert(LocalDate.now()));
+            String benefitType;
+            if (sscsCaseData.getAppeal().getBenefitType().getCode().equalsIgnoreCase("PIP")) {
+                benefitType = "Taliad Annibyniaeth Personol (PIP)";
```

Comment:
Could we not have the Welsh benefit types in a separate file rather than hardcoding them here? Just in case we need to use somewhere else

Reply:
Have refactored this code to externalise the translations as suggested